spacy_experimental/span_suggesters/chunk_suggester.py

```diff
@@ -0,0 +1,55 @@
+from typing import List, Optional, Iterable, cast
+from thinc.api import get_current_ops, Ops
+from thinc.types import Ragged, Ints1d
+from spacy.tokens import Doc
+from spacy.util import registry
+from spacy.pipeline.spancat import Suggester
+from .merge_suggesters import merge_suggestions
+
+
+@registry.misc("experimental.ngram_chunk_suggester.v1")
```

Comment:
Because we have them as entry points in `setup.cfg`, we can remove the decorators from these functions (also ensure that the names don't go out-of-sync, which they have here)